src/cpp/fastdds/subscriber/DataReaderImpl.hpp

```diff
@@ -248,21 +248,19 @@ class DataReaderImpl
     ReturnCode_t get_requested_incompatible_qos_status(
             RequestedIncompatibleQosStatus& status);
 
-    /* TODO
-       bool get_sample_lost_status(
-            fastrtps::SampleLostStatus& status) const;
-     */
+    ReturnCode_t get_sample_lost_status(
```

Comment:
We are now requiring doxydoc on internal methods, please add it.

Reply:
Done on b09d55f7c1d1f39a1fb345126a46acf73e4c6845